packages/create-twilio-function/README.md

```diff
@@ -2,7 +2,7 @@
 
 A command line tool to setup a new [Twilio Function](https://www.twilio.com/docs/api/runtime/functions) with local testing using [`twilio-run`](https://github.com/twilio-labs/twilio-run).
 
-[![Build Status](https://travis-ci.com/twilio-labs/create-twilio-function.svg?branch=master)](https://travis-ci.com/twilio-labs/create-twilio-function) [![Maintainability](https://api.codeclimate.com/v1/badges/e6f9eb67589927df5d72/maintainability)](https://codeclimate.com/github/twilio-labs/create-twilio-function/maintainability)
+[![Maintainability](https://api.codeclimate.com/v1/badges/e6f9eb67589927df5d72/maintainability)](https://codeclimate.com/github/twilio-labs/serverless-toolkit/maintainability)
```

Comment:
Hmm... how does CodeClimate work with a monorepo? This badge may just not work any more. Perhaps, in another issue, we should align all repo badges across the packages?

Reply:
Good point. I removed it for now and added #184 as a new issue